lib/cli.bash

```diff
@@ -18,7 +18,43 @@ function _omb_cmd_reload {
   echo 'Not yet implemented'
 }
 function _omb_cmd_theme {
-  echo 'Not yet implemented'
+  case "$1" in
+  list)
+      local -a available_themes
+      _comp_cmd_omb__get_available_themes
+      for theme in "${available_themes[@]}"; do
+        echo "$theme"
+      done
+    ;;
+  use)
+    local theme=$2
+    if [[ -z "$theme" ]]; then
+      echo 'Usage: omb theme use <theme>'
+      return 2
+    fi
+    local -a available_themes
+    _comp_cmd_omb__get_available_themes
+    for i in "${available_themes[@]}"; do
+      if [ "$i" == "$theme" ]; then
+        _omb_module_require_theme "$theme"
+        return 0
+      fi
+    done
+    echo "Theme '$theme' not found"
```

Comment:
```suggestion
    _omb_util_print "Theme '$theme' not found"
```

We avoid using `echo` because `echo` is affected by the setting `shopt -s xpg_echo` and is not trustable.